packages/hash/frontend/src/blocks/page/createSuggester/BlockSuggester.tsx

```diff
@@ -110,7 +110,7 @@ export const BlockSuggester: VFC<BlockSuggesterProps> = ({
           </Box>
         ) : null
       }
-      itemKey={({ meta, variant }) => `${meta.name}/${variant.name}`}
+      itemKey={({ meta, variant }) => `${meta.name}/${variant.displayName}`}
```

Comment:
@CiaranMn did this change I think – any idea why?

Reply:
I changed it from `displayName` to `description` (I think because there were duplicate `name` across some variants, but I can't now find them), and then it got changed back to `displayName` again in a merge commit.

Reply:
`name`/`name` seems fine

Reply:
Cool – will fix that.